### PR TITLE
flux: add livecheck block for resource

### PR DESCRIPTION
**Pull request by mohammadzainabbas**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Similar to https://github.com/Homebrew/homebrew-core/pull/112155, now we will fetch `pkg-config-wrapper` livecheck info from upstream repo version in `go.mod` of `flux`.

@nandahkrishna kindly, have a look.

### CODE REVIEW

**Review by samford — COMMENTED** [review]
Same as the `influxdb` PR (#112155), checking the `go.mod` file from the `master` branch isn't appropriate, as it's not tied to a release version. This is another instance where we need to check the `go.mod` file for the latest version from main formula livecheck, as described in https://github.com/Homebrew/homebrew-core/pull/112155#issuecomment-1355090824.

As it stands, we can't create an appropriate resource `livecheck` block for`pkg-config-wrapper` until we're able to reference the latest version in the URL. Nanda has some forthcoming work for this, so we'll have to wait until that's merged and available.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**
